site/_data/download_types.yaml

```diff
@@ -1,48 +1,41 @@
 - name: Microsoft Azure
   download_platform: azure
   ext: vhd
-  size_prev: 3.9 GB
-  size_stable: 3.7 GB
-  size_devel: 3.2 GB
+  size_stable: 3.8 GB
+  size_devel: 2.8 GB
 
 - name: Microsoft SCVMM
   download_platform: hyperv
   ext: vhd
-  size_prev: 3.9 GB
-  size_stable: 3.8 GB
-  size_devel: 3.2 GB
+  size_stable: 3.9 GB
+  size_devel: 2.8 GB
 
 - name: OpenStack
   download_platform: openstack
   ext: qc2
-  size_prev: 1.5 GB
   size_stable: 1.4 GB
-  size_devel: 1.2 GB
+  size_devel: 1.1 GB
 
 - name: oVirt
   download_platform: ovirt
   ext: ova
-  size_prev: 1.4 GB
   size_stable: 1.3 GB
-  size_devel: 1.1 GB
+  size_devel: 1.0 GB
 
 - name: QEmu/KVM
   download_platform: openstack
   ext: qc2
-  size_prev: 1.5 GB
   size_stable: 1.4 GB
-  size_devel: 1.2 GB
+  size_devel: 1.1 GB
 
 - name: Red Hat Enterprise Virtualization
   download_platform: ovirt
   ext: ova
-  size_prev: 1.4 GB
   size_stable: 1.3 GB
-  size_devel: 1.1 GB
+  size_devel: 1.0 GB
 
 - name: VMware vSphere
   download_platform: vsphere
   ext: ova
-  size_prev: 1.5 GB
-  size_stable: 1.3 GB
-  size_devel: 1.1 GB
+  size_stable: 1.4 GB
+  size_devel: 1.0 GB
```

Comment:
I think they are useful for providing a good idea of how large the downloads are. I'll take care of updating them every release for now and I'm also looking at a way to automate it. As for devel, the changes daily are probably minor, so I'll check and update occasionally until the automate system is in place. Main goal is to provide a guide, and as you know, different systems calculate sizes differently so it's also subject to interpretation :grin: